### PR TITLE
Authentication and readme

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello,

I have added Forgot Password, Profile page and updated README.md.

### Forgot Password Page

image

### After Send Email Button Click

image

### Profile Page

image

### Address Tab

image

*I have also refactored the headers and footers for all the pages. (You can ignore these while reviewing)*

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -67,12 +86,12 @@
...
-        <div class="carousel-container fade">
-          <div class="container fade">
+        <div class="carousel-container">
+          <div class="container">
             <img class="carousel-img" src="/images/carousel-1.jpg" />
           </div>
 
-          <div class="container fade">
+          <div class="container">
             <img class="carousel-img" src="/images/carousel-3.jpg" />
```
Comment (vrinda-mahajan) [inline index.html]: Add the alt attribute to the image tags.

File: script.js [hunk 2]
```diff
@@ -1,40 +1,94 @@
...
+    addressBtn.addEventListener("click", () => {
+        profileSection.style.display = "none"
+        addressSection.style.display = "block";
+    })
+
+    profileBtn.addEventListener("click", () => {
+        profileSection.style.display = "block"
+        addressSection.style.display = "none";
+    })
+}
```
Comment (vrinda-mahajan) [inline script.js:94]: It is a good practice to add an extra line at the end of each file.

File: styles.css [hunk 3]
```diff
@@ -115,16 +106,32 @@ footer {
...
+    .myhamburger {
+        display: block;
+    }
+}
+
+@media screen and (max-width: 480px) {
+    .sub-navigation {
+        display: none;
+    }
 }
```
Comment (vrinda-mahajan) [inline styles.css:137]: Here also add a new line.

File: pages/product.html [hunk 4]
```diff
@@ -18,44 +18,63 @@
       >
```
Comment (vrinda-mahajan) [inline pages/product.html:18]: Screenshot (68)
Your filter sidebar is overlapping your footer. Please look into it.

**Review by vrinda-mahajan — COMMENTED** [review]
Your code is clean and UI is also looking superb. Just added a few suggestions.

File: pages/product.html (hunk 4 above)
Comment (ApurvaSawant11) [reply pages/product.html:18]: Fixed it. Thanks

File: styles.css (hunk 3 above)
Comment (ApurvaSawant11) [reply styles.css:137]: Prettier file formatter removes blank lines at the end

File: script.js (hunk 2 above)
Comment (ApurvaSawant11) [reply script.js:94]: Prettier file formatter removes blank lines at the end

File: index.html (hunk 1 above)
Comment (ApurvaSawant11) [reply index.html]: Done. Thanks

**Review by vrinda-mahajan — APPROVED** [review]
Good to merge.

**Review by jalajcodes — COMMENTED** [review]
Loved your UI. Good to merge

About prettier, I use it too and it adds newline at the end of every file. I can see in your code html files have newline at end, but css files don't, maybe your prettier is not configured for css files.

**Comment by ApurvaSawant11** [comment]
> Loved your UI. Good to merge
> 
> About prettier, I use it too and it adds newline at the end of every file. I can see in your code html files have newline at end, but css files don't, maybe your prettier is not configured for css files.

Okay... I checked and couldn't find anything. Will look into it later. Thanks!

**Pull request merged**